pages/about/finances.md

```diff
@@ -6,15 +6,17 @@ menubar: about
 set_last_modified: true
 ---
 
-### Open Collective Foundation Member 
+### Community Initiatives
```

Comment:
Do we need to put "member" or anything like that?

Reply:
How is that?